src/main.rs

```diff
@@ -161,12 +167,47 @@ fn main() -> Result<()> {
 
         let parameters = Parameters::new();
 
-        if matches.subcommand_matches("list").is_some() {
-            let list = parameters.get_parameter_names(org_id, environments.get_id(org_id, env)?)?;
-            if list.is_empty() {
-                println!("There are no parameters in your account.")
+        if let Some(matches) = matches.subcommand_matches("list") {
+            let values = matches.is_present("values");
+            if !values {
+                let list =
+                    parameters.get_parameter_names(org_id, environments.get_id(org_id, env)?)?;
+                if list.is_empty() {
+                    println!("There are no parameters in your account.")
+                } else {
+                    println!("{}", list.join("\n"))
+                }
             } else {
-                println!("{}", list.join("\n"))
+                let fmt = matches.value_of("format").unwrap();
+                let env_id = environments.get_id(org_id, env)?;
+                let ct_vars = parameters.get_parameter_details(org_id, env_id)?;
```

Comment:
This returns a different set of keys than the `list` command without the `--values` flag because it strips out keys that would lead to invalid env var values. I don't think it's necessary to strip any keys here.